seed/seed.json

```diff
@@ -2939,47 +2939,19 @@
             "filter": {
                 "channel": [
                     "NIGHTLY",
-                    "BETA"
-                ],
-                "min_version": "125.*",
-                "platform": [
-                    "WINDOWS",
-                    "MAC",
-                    "LINUX",
-                    "ANDROID"
-                ]
-            },
-            "name": "BraveRoundTimeStampsStudy"
-        },
-        {
-            "experiments": [
-                {
-                    "feature_association": {
-                        "disable_feature": [
-                            "BraveRoundTimeStamps"
-                        ]
-                    },
-                    "name": "Disabled",
-                    "probability_weight": 25
-                },
-                {
-                    "name": "Default",
-                    "probability_weight": 75
-                }
-            ],
-            "filter": {
-                "channel": [
+                    "BETA",
                     "RELEASE"
                 ],
                 "min_version": "125.*",
+                "max_version": "130.1.72.75",
```

Comment:
The feature was disabled here: https://github.com/brave/brave-core/pull/25877#issuecomment-2402130204